src/ralph/export_to_ng/publishers.py

```diff
@@ -194,3 +195,34 @@ def sync_virtual_server_to_ralph3(sender, instance=None, created=False, **kwargs
         'parent_id': asset.id if asset else None,
         'custom_fields': _get_custom_fields(instance),
     }
+
+
+@ralph3_sync(Network, topic='sync_ipaddress_to_ralph3')
+def sync_gateway_ipaddress_to_ralph3(sender, instance=None, created=False, **kwargs):
+    if not instance.gateway:
+        return {}
+    return {
+        'id': None,
+        'address': instance.gateway,
+        'device_id': None,
+        'hostname': None,
+        'is_management': False,
+        'is_gateway': True,
+        'dhcp_expose': False
+    }
+
+
+@ralph3_sync(IPAddress)
+def sync_ipaddress_to_ralph3(sender, instance=None, created=False, **kwargs):
+    return {
+        'id': instance.id,
+        'address': instance.address,
+        'device_id': instance.device_id,
```

Comment:
it wont be so simple :) IMO we should return here more information, like type (asset, virtual etc) and then it's id. In case of physical device, return its asset id